tools/marketplace/helper/bundleHelper.go

```diff
@@ -0,0 +1,307 @@
+/*
```

Comment:
filename not consistent with others in this repo.  Should it be `bundle-helper.go`?

Or if this is an exact copy of the file in KubeVirt, maybe mention that?

Reply:
changing file name since the file is not the exact copy of the file in KubeVirt